### ISSUE TITLE
Why not using BERT or other PrLMs as the baseline model?

### ISSUE THREAD

**Issue by LilyNLP** [nitaytech/DoCoGen#1]
If I understand correctly, DoCoGen is an excellent method to augment the data for domain adaptation and thus we can apply any model to the augmented data. I just wonder why you did not use BERT on the augmented data? It seems that BERT can give very high accuracy in UDA scenarios. It seems promising to apply BERT on the DoCoGen data to achieve a higher accuracy?

**Comment by nitaytech** [comment]
You are absolutely right, and this is exactly what we did in our paper (https://arxiv.org/abs/2202.12350). 

We didn't include the fine-tuning step of the NLP task classifier (on the original data + the augmented data) in this repo, because we didn't want to limit other researchers to use a particular NLP task classifier (such as BERT) or to a specific training pipeline (e.g. you can use the augmented data to facilitate self-training and pseudo-labeling). 

We have shown in the paper how the augmented data can be used for improving another UDA algorithm -- PERL (https://arxiv.org/abs/2006.09075).
